### ISSUE TITLE
GLES2 SCREEN_TEXTURE gives a black screen

### ISSUE THREAD

**Issue by arnoldochavez** [godotengine/godot#21899]
**Godot version:**
Godot_v3.1-alpha1

**OS/device including version:**
Windows 10 / Intel HD Graphics:
Driver Version: 8.15.10.2900
Operating System: (6.2.9200)
Default Language: English (United States)
DirectX* Version: 10.1
Physical Memory: 5941 MB
Minimum Graphics Memory: 32 MB
Maximum Graphics Memory: 1696 MB
Graphics Memory in Use: 230 MB
Processor: Intel64 Family 6 Model 37 Stepping 5
Processor Speed: 2667 MHz
Vendor ID: 8086
Device ID: 0046
Device Revision: 02

**Issue description:**
Using a SCREEN_TEXTURE in a shader gives a black screen.

![shaders](https://user-images.githubusercontent.com/33552997/45266274-e3905500-b415-11e8-9f97-ff73976732d2.gif)

**Steps to reproduce:**
Put a texture rect with an image and another one in front with the shader using:
```
shader_type canvas_item;

void fragment() {
	COLOR.rgb= textureLod(SCREEN_TEXTURE,SCREEN_UV, 0.0).rgb;
}
```

**Minimal reproduction project:**
The Screen-Space Shader project
[ScreenSpaceShaders.zip](https://github.com/godotengine/godot/files/2364397/ScreenSpaceShaders.zip)

**Comment by 0nepixel** [comment]
GLES2 doesn't work by the moment

**Comment by akien-mga** [comment]
@ni-code That's a different issue, see #15998.

**Comment by Nobelix** [comment]
Does the same thing for me in 3.1 beta1 with my GTX1050.

**Comment by Th0r12356** [comment]
Tested ScreenSpaceShaders and some other shaders with SCREEN_TEXTURES in gles2. Everything works =) I think this issue is fixed by reduz in 7fa41f1. I really, really appreciate it!

**Comment by akien-mga** [comment]
Thanks, closing as fixed by 7fa41f1.
